Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1614,7 +1614,7 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
       if (mT0 != 0.0) {
         int64_t numHistograms =
             static_cast<int64_t>(m_ws->getNumberHistograms());
-        PARALLEL_FOR1(m_ws)
+        PARALLEL_FOR_IF(m_ws->threadSafe())
```

Comment:
`m_ws` is an `EventWorkspaceCollection` which does NOT inherit from `DataItem` and `m_ws->threadsafe() will always return true. Is there any reason to keep this check?